package.json

```diff
@@ -11,7 +11,7 @@
         "node-uuid": "~1.4.0",
         "semver": "~1.1.4",
         "xml-mapping": "~1.2.0",
-        "kettle": "git://github.com/fluid-project/kettle.git#5a2e8b8fb5b198b266d6b7b7d0d9eb94185a3759"
+        "kettle": "git://github.com/fluid-project/kettle.git#27a3bf7cfd51f62e66b45bc1b7bae9f407052586"
```

Comment:
Not sure why this diff is shown here - it has already been merged in with: https://github.com/GPII/universal/commit/222ca7ea4f492b0ea2885ce71668670a4f095056